iiif_pipeline/derivatives.py

```diff
@@ -130,4 +119,8 @@ def ocr_pdf(identifier, pdf_dir):
         pdf_dir (str): Directory in which to save the PDF file.
     """
     pdf_path = "{}.pdf".format(os.path.join(pdf_dir, identifier))
-    ocrmypdf.ocr(pdf_path, pdf_path, optimize=0, output_type="pdf")
+    subprocess.run(["/usr/local/bin/ocrmypdf",
+                    pdf_path, pdf_path,
+                    "--output-type", "pdf",
+                    "--optimize", "3",
```

Comment:
Looks like the optimize level changed from 3 to 0; it should probably remain at 0.